Source/common/CEL.h

```diff
@@ -0,0 +1,83 @@
+/// Copyright 2025 North Pole Security, Inc.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///    http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+
+#ifndef SANTA__COMMON__CEL_H
+#define SANTA__COMMON__CEL_H
+
+#include "Source/common/CEL.pb.h"
+
+#include <memory>
+
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
+#include "sync/v1.pb.h"
+
+// CEL headers have warnings and our config turns them into errors.
+// For some reason these can't be disabled with --per_file_copt.
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wshorten-64-to-32"
+#include "cel/expr/checked.pb.h"
+#include "compiler/compiler.h"
+#include "eval/public/cel_expression.h"
+#pragma clang diagnostic pop
+
+namespace santa {
+
+class CELEvaluator {
+ public:
+  // Initialize the evaluator. This must be called before any other methods as
+  // it initializes the CEL compiler.
+  absl::Status Initialize();
```

Comment:
It doesn't seem like this class has any useful functionality if not `Initialized`. It would be cleaner API to add a static factory method that creates the compiler, and create a non-default constructor to move the compiler into.